src/renderer/components/+custom-resources/crd-resource-details.tsx

```diff
@@ -24,18 +24,18 @@ import "./crd-resource-details.scss";
 import React from "react";
 import jsonPath from "jsonpath";
 import { observer } from "mobx-react";
-import { computed, makeObservable } from "mobx";
 import { cssNames } from "../../utils";
 import { Badge } from "../badge";
 import { DrawerItem } from "../drawer";
 import type { KubeObjectDetailsProps } from "../kube-object";
-import { crdStore } from "./crd.store";
 import { KubeObjectMeta } from "../kube-object/kube-object-meta";
 import { Input } from "../input";
 import type { AdditionalPrinterColumnsV1, CustomResourceDefinition } from "../../api/endpoints/crd.api";
 import { parseJsonPath } from "../../utils/jsonPath";
+import type { KubeObject, KubeObjectMetadata, KubeObjectStatus } from "../../api/kube-object";
 
-interface Props extends KubeObjectDetailsProps<CustomResourceDefinition> {
+interface Props extends KubeObjectDetailsProps<KubeObject> {
+  crd: CustomResourceDefinition;
```

Comment:
I changed to passing in the crd (which should be the parent crd of `object`) because the single use of this component already needs to have it.